apps/docsite/src/components/BundleSizeTable.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react'
+
+const BundleSizeTable = () => {
+    const [bundleSizeData, setBundleSizeData] = useState();
+    const createTableFromJSON = (JSONData) => {
+        JSONData.map((item, index) => {
+            console.log(item)
+        })
+    }
+    useEffect(() => {
+        const fetchData = async () => {
+            try {
+                const response = await fetch("https://raw.githubusercontent.com/Shubhabrata08/fluentui/add-monosize-measure-json/packages/react-charting/monosize.json");
+                if (!response) {
```

Comment:
how are you generating this json

Reply:
Running yarn monosize measure command in charting package directory

Reply:
We will schedule the generation by a cron job